[2023-07-30 22:05:59,057] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-05-01T00:00:00+00:00 [queued]>
[2023-07-30 22:05:59,233] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-05-01T00:00:00+00:00 [queued]>
[2023-07-30 22:05:59,234] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:59,234] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-30 22:05:59,234] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:59,526] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): dag_envio> on 2023-05-01 00:00:00+00:00
[2023-07-30 22:05:59,585] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_smtp_email_automatico', 'dag_envio', 'scheduled__2023-05-01T00:00:00+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/dag_smtp_email.py', '--cfg-path', '/tmp/tmpe307ca6w', '--error-file', '/tmp/tmpfxfund9g']
[2023-07-30 22:05:59,586] {standard_task_runner.py:80} INFO - Job 238: Subtask dag_envio
[2023-07-30 22:05:59,568] {standard_task_runner.py:52} INFO - Started process 1371 to run task
[2023-07-30 22:06:00,111] {task_command.py:371} INFO - Running <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-05-01T00:00:00+00:00 [running]> on host b0e2d990d7ae
[2023-07-30 22:06:00,487] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ariel Feldman
AIRFLOW_CTX_DAG_ID=dag_smtp_email_automatico
AIRFLOW_CTX_TASK_ID=dag_envio
AIRFLOW_CTX_EXECUTION_DATE=2023-05-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-01T00:00:00+00:00
[2023-07-30 22:06:02,037] {logging_mixin.py:115} INFO - Connection unexpectedly closed
[2023-07-30 22:06:02,040] {logging_mixin.py:115} INFO - Failure
[2023-07-30 22:06:02,040] {python.py:173} INFO - Done. Returned value was: None
[2023-07-30 22:06:02,079] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_smtp_email_automatico, task_id=dag_envio, execution_date=20230501T000000, start_date=20230730T220559, end_date=20230730T220602
[2023-07-30 22:06:02,209] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-30 22:06:02,317] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
